aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -133,53 +133,38 @@ class QConv2dInt8 final : public c10::OperatorKernel {
     float act_scale = act.q_scale();
     int32_t act_zero_point = act.q_zero_point();
 
-    const int32_t* bias_ptr = nullptr;
-    at::Tensor qbias;
+    const float* bias_ptr = nullptr;
+    at::Tensor bias;
     if (pack_ptr.bias.has_value()) {
-      at::Tensor bias = pack_ptr.bias.value();
-      // Temporary: Quantize bias
-      if (pack_ptr.q_scheme == kPerTensorAffine) {
-        qbias = at::quantize_linear(
-            at::dequantize(bias), pack_ptr.w_scale[0] * act_scale, 0, kQInt32);
-      } else if (pack_ptr.q_scheme == kPerChannelAffine) {
-        std::array<int64_t, 1> arr{0};
-        IntArrayRef axis(arr.data(), 1);
-        at::Tensor bias_scale = at::ones({K}, at::dtype(at::kDouble));
-        at::Tensor bias_zp = at::zeros({K}, at::dtype(at::kLong));
-        for (int i = 0; i < K; ++i) {
-          bias_scale.data_ptr<double>()[i] = pack_ptr.w_scale[i] * act_scale;
-        }
-        qbias = quantize_linear_per_channel_cpu(
-            at::dequantize(bias), bias_scale, bias_zp, axis, kQInt32);
-      } else {
-        qbias = bias;
-        TORCH_CHECK(false, "Unsupported quantization scheme.")
-      }
-      TORCH_CHECK(qbias.dim() == 1, "bias should be a vector (1D Tensor)");
+      bias = pack_ptr.bias.value();
+      bias = bias.contiguous();
+      TORCH_CHECK(bias.dim() == 1, "bias should be a vector (1D Tensor)");
       TORCH_CHECK(
-          qbias.size(0) == K,
+          bias.size(0) == K,
           "bias should have K elements: " + std::to_string(K));
-      auto bias_contig = qbias.contiguous();
-      bias_ptr =
-          reinterpret_cast<int32_t*>(bias_contig.data_ptr<c10::qint32>());
+      bias_ptr = bias.data_ptr<float>();
```

Comment:
I'd add a nice TORCH_CHECK that the input is unquantized with a human-understandable message. data_ptr would do one for you but it's nicer to call out that we expect bias in fp32

Reply:
Added a check.